modules/core/src/index.js

```diff
@@ -154,7 +154,11 @@ export {default as diffuse} from './shadertools/src/modules/diffuse/diffuse';
 export {default as pbr} from './shadertools/src/modules/pbr/pbr';
 
 // TODO: move to gpgpu module.
-export {getHistoPyramid} from './utils/histo-pyramid';
+export {
+  buildHistopyramidBaseLevel as _buildHistopyramidBaseLevel,
```

Comment:
core is growing, time to create the new gpgpu module!

Reply:
Will do this in separate PR.